test/functional/address_types.py

```diff
@@ -104,10 +112,32 @@ def test_address(self, node, address, multisig, typ):
             # Unknown type
             assert(False)
 
+    def test_change_output_type(self, node_sender, destinations, expected_type):
+        txid = self.nodes[node_sender].sendmany("",  dict.fromkeys(destinations, 0.001))
+        raw_tx = self.nodes[node_sender].getrawtransaction(txid)
+        tx = self.nodes[node_sender].decoderawtransaction(raw_tx)
+
+        # Make sure the transaction has change:
+        assert_equal(len(tx["vout"]), len(destinations) + 1)
+
+        output_addresses = list(map(lambda vout: vout['scriptPubKey']['addresses'][0], tx["vout"]))
+
+        # Make sure the destinations are included, and remove it:
+        for destination in destinations:
+            output_addresses.pop(output_addresses.index(destination))
+
+        assert_equal(len(output_addresses), 1)
+
```

Comment:
style-nit: heh, you are excessively using new lines in python code

Reply:
Removed a few blank lines...